packages/syft/src/syft/service/user/user.py

```diff
@@ -173,7 +173,8 @@ def user_create_to_user() -> List[Callable]:
         validate_email,
         hash_password,
         generate_key,
-        default_role(ServiceRole.GUEST),
+        # TODO: Fix this by passing it from client & verifying it at server
+        default_role(ServiceRole.DATA_SCIENTIST),
```

Comment:
Currently, we use the register function for registering data scientist users, but they are incorrectly registered with the role ServiceRole Guest.

Since we mainly , use the register functionality for registering data scientists.
and by the default PR: https://github.com/OpenMined/PySyft/pull/7715
Automatic registration is toggled off for non admin users.

So we set the default service role as data scientist for now. But in later PR's we could make the client submit the role.